README.md

```diff
@@ -131,17 +131,18 @@ $ sudo td-agent-gem install fluent-plugin-geoip
   </record>
 
   # Settings for tag
-  remove_tag_prefix access.
-  tag               geoip.${tag}
+  tag               geoip.${tag[1]}
 
   # To avoid get stacktrace error with `[null, null]` array for elasticsearch.
   skip_adding_null_record  true
 
   # Set log_level for fluentd-v0.10.43 or earlier (default: warn)
   log_level         info
 
-  # Set buffering time (default: 0s)
-  flush_interval    1s
+  <buffer tag>
```

Comment:
For example,

* time
* tag
* record key to use record value

This is undocumented for now.
Maybe hostname is not available. If you want to separate chunk by hostname you can use filter record transformer to inject hostname in records.